src/services/color/color-sets/micro-focus-color-set.ts

```diff
@@ -64,8 +64,10 @@ export const MICRO_FOCUS_COLOR_SET = {
     'chart4': '#ffce00',
     'chart5': '#eb23c2',
     'chart6': '#ba47e2',
+    'info': '00abf3',
```

Comment:
```suggestion
    'info': '#00abf3',
```
Maybe should fix `critical` there as well.